Automattic/jetpack
# Move with-has-warning-is-interactive-class-names folder to shared-extension-utils package

Updated imports in jetpack plugin





Fixes #

#### Changes proposed in this Pull Request:

* Moved with-has-warning-is-interactive-class-names folder to shared-extension-utils package. Updated imports for the same in jetpack plugin
* Added .eslintrc.cjs file in shared-extension-utils for parsing jsx files.

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?
No



#### Testing instructions:
Make sure CI is happy
`pnpm build-extensions` command should run without any errors in Jetpack plugin folder

* Go to '..'
*
